ramalama

```diff
@@ -333,6 +333,48 @@ def pull_cli(ramalama_store, args, port):
 funcDict["pull"] = pull_cli
 
 
+def push_oci(model, ramalama_store):
+    # Remove the prefix and extract target details
+    target = re.sub(r'^oci://', '', model)
+    registry, reference = target.split('/', 1)
+    registry, reference = ("docker.io", target) if "." not in registry else (
```

Comment:
We should not force docker.io.  Users need to fully specify on push.

Reply:
We aren't forcing indeed,
this is more to comply with the fact if I indicate `my-artifact:v1` as a reference,
it is intended to be `docker.io/library/my-artifact:v1`.

I believe most user flow however are in the complete form, such as `quay.io/mmortari/my-artifact:v1` anyway.

If the reference convention is not desired, we can always make it display an error message and fail, for instance ~"you must supply OCI registry in your target reference".

wdyt?

Reply:
I would prefer the latter.

Reply:
@rhatdan ack 👍

Reply:
I cross checked. This is for consistency of Pull semantic for OCI reference, where if not specified (eg.: `mmortari/model:v1`) is intended as coming from docker.io.

Reply:
That is the way Docker works but not Podman.  Podman allows you to set a priority on container registries.  We should not hard code a preference for storing at any container registry.  Especially not a fully free one.

Reply:
TIL, and thank you for the explanation; will adapt to it ! 👍

Reply:
👉  290a52b536c1ee693b760551f7a8388ef0bd4b33